### ISSUE TITLE
I have problems with admob blueprinth

### ISSUE THREAD

**Issue by AlexysNeith** [IamKingWaiMark/GoogleAdmobBlueprint#7]
I would like to know what I should do. I had these problems when importing it for Android. I did the entire video procedure from where it starts to where it ends. I would like to know if I have to activate something else or deactivate it. I am using Unreal engine 5.3 That only happens every time I activate the plugin, but if I deactivate it, it copies the project normally. 
![Imagen de WhatsApp 2024-05-18 a las 17 44 32_8cfac8a6](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/b1eb5919-4be3-4c86-b060-34999ae3befc)

![Uploading 20240518_154259.jpg…]()

**Comment by IamKingWaiMark** [comment]
Hello, does the package name match in the Google services file? Make sure it is the case is also the same.

**Comment by AlexysNeith** [comment]
Yes, in fact I have done all the steps from beginning to end and I still get the same error
![Uploading Captura de pantalla 2024-05-19 114750.png…]()

**Comment by IamKingWaiMark** [comment]
Hello, can you attach the google service.json file?

**Comment by AlexysNeith** [comment]
![Captura de pantalla 2024-05-19 121057](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/a3bc84cb-37ec-4aff-8ea5-bb7b600077d5)
There is this image, do you know what the problem could be?

**Comment by AlexysNeith** [comment]
.

**Comment by AlexysNeith** [comment]
please help

**Comment by IamKingWaiMark** [comment]
Hello, can you also attach the whole logs?

**Comment by AlexysNeith** [comment]
![Captura de pantalla 2024-06-06 233419](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/3d05cda3-2e47-4bdd-8d94-30867da675ab)
![Captura de pantalla 2024-06-06 233430](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/31e88a12-0f13-453b-983f-e0b6842e40ff)
![Captura de pantalla 2024-06-06 233447](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/8c3c2f05-5a02-449f-b655-2a8e630162f6)
![Captura de pantalla 2024-06-06 233500](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/44dfc5a6-e050-4a8f-b21f-94bed21c1301)
![Captura de pantalla 2024-06-06 233515](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/da8a6961-d5a9-4258-a171-51c389556e34)
![Captura de pantalla 2024-06-06 233530](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/44693d26-1c39-486d-8bc6-e86e8c808139)

**Comment by AlexysNeith** [comment]
[google-services.json](https://github.com/user-attachments/files/15709941/google-services.json)

**Comment by AlexysNeith** [comment]
I just switched to unreal engine 5.2 and I still get an error

**Comment by AlexysNeith** [comment]
![Uploading Captura de pantalla 2024-06-06 234848.png…]()

**Comment by AlexysNeith** [comment]
![Captura de pantalla 2024-06-06 234848](https://github.com/IamKingWaiMark/GoogleAdmobBlueprint/assets/141919040/b394a7b6-1951-4fb8-80e4-534438e3060a)

**Comment by AlexysNeith** [comment]
Forget it, it already worked for me, it is only compatible with unreal engine 5.2
